Library/Homebrew/cmd/update.sh

```diff
@@ -364,6 +389,10 @@ EOS
     pull "$DIR"
   done
 
+  if [[ -n "$HOMEBREW_VERBOSE" ]]
+  then
+    oh1 "Summarizing Homebrew update"
+  fi
```

Comment:
I would suggest that this message end with a colon. Also, I hope that this last message will be shown whether or not the user has any taps at all, and whether or not the taps are git repos, and that there not be a "heading-less" emission of "Already up-to-date" in any of those same circumstances.

I still think the best solution is to drop this header, but make sure the reported message isn't identical to `Already up-to-date`.  Maybe `Homebrew installation is already up to date` or something of that nature.